arch/conduwuit.service

```diff
@@ -0,0 +1,61 @@
+[Unit]
+Description=conduwuit Matrix homeserver
+After=network.target
+Documentation=https://conduwuit.puppyirl.gay/
+
+[Service]
+DynamicUser=yes
+Type=notify
+
+AmbientCapabilities=
+CapabilityBoundingSet=
+
+DevicePolicy=closed
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+ProcSubset=pid
+ProtectClock=yes
+ProtectControlGroups=yes
+ProtectHome=yes
+ProtectHostname=yes
+ProtectKernelLogs=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
+ProtectProc=invisible
+ProtectSystem=strict
+PrivateDevices=yes
+PrivateMounts=yes
+PrivateTmp=yes
+PrivateUsers=yes
+PrivateIPC=yes
+RemoveIPC=yes
+RestrictAddressFamilies=AF_INET AF_INET6 AF_UNIX
+RestrictNamespaces=yes
+RestrictRealtime=yes
+RestrictSUIDSGID=yes
+SystemCallArchitectures=native
+SystemCallFilter=@system-service @resources
+SystemCallFilter=~@clock @debug @module @mount @reboot @swap @cpu-emulation @obsolete @timer @chown @setuid @privileged @keyring @ipc
+SystemCallErrorNumber=EPERM
+StateDirectory=matrix-conduit
```

Comment:
```suggestion
StateDirectory=conduwuit
```

Reply:
I have to make sure it doesn't brick existing installation in the PKGBUILD before this can be changed

Reply:
If `StateDirectory=` can be set multiple times, I'm okay with having both there. If it helps.

Reply:
I'm testing with new .install script, which will automatically move directories into the right spot